README.md

```diff
@@ -250,3 +250,5 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [haste-server](/software/haste-server)
 ### RabbitMQ
 * [rabbitmq](/software/rabbitmq)
+### Language-Tool
```

Comment:
The list is in alphabetical order. Put this above RabbitMQ. 

The name is LanguageTool, as seen on their website, not Language-Tool. Please correct this to match in both readmes you've created.